src/bin/busd.rs

```diff
@@ -26,6 +28,48 @@ struct Args {
     #[clap(long)]
     #[arg(value_enum, default_value_t = AuthMechanism::External)]
     auth_mechanism: AuthMechanism,
+
+    /// File descriptor to which readiness notifications are sent.
+    ///
+    /// Once the server is listening to connections on the specified socket, it will print
+    /// `READY=1\n` into this file descriptor and close it.
+    ///
+    /// This readiness notification mechanism which works on both systemd and s6.
+    #[clap(long)]
+    ready_fd: Option<i32>,
```

Comment:
should be behind a unix cfg. It doesn't work on Windows. Perhaps it's better if we also handle this via `--print-pid` like dbus-daemon? Then we can handle it in a cross-platform way w/o any cfgs invovled?